invoices/xml/__init__.py

```diff
@@ -36,7 +36,7 @@ def _generate_header(invoice: Invoice) -> XMLDict:
                     "IdPaese": sender.country_code,
                     "IdCodice": sender.code,
                 },
-                "ProgressivoInvio": invoice.invoice_number,
+                "ProgressivoInvio": 1,
```

Comment:
Why 1 hard coded?

Reply:
from what I have understood this number needs to be increased every time we try to send the XML to the SdI. For now we are only going to send the invoice once, as we will use an external API/service (Aruba) to send the invoices we will use that servive for fixing and sending the invoices again.